### DISCUSSION TITLE
javascript runners

### DISCUSSION THREAD

**Discussion by Sam-hacks1** [3kh0/ext-remover#1365] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
are there any websites or ways to run javascript

**Comment by Charlieis1nsan3** [comment, marked as answer]
Ublock, or Snap&Run, any others?   Nope!

**Reply by Charlieis1nsan3** [reply]
or bookmarklets

**Reply by Sam-hacks1** [reply]
is snap and read the same thing because that's what I have

**Comment by S-PScripts** [comment]
providing javascript: isn't blocked by policy, you could downgrade (idk when chromeos patched javascript probably around v11(X) to use it again
